.travis.yml

```diff
@@ -5,9 +5,13 @@ go:
 
 services:
   - docker
+  - postgresql
 
 sudo: false
 
+before_script:
+  - psql -c 'create database docker_test;' -U postgres
+
```

Comment:
sure for the `createdb` command the last option has be to be db name, but for `psql -c` you are just sending a sql query so it doesn't matter (i don't think, don't have a windows machine which i care to test it on)